### PR TITLE
PARQUET-519: Remove last of suppressed compiler warnings

### PR DESCRIPTION
**Pull request by wesm**

### CODE REVIEW

File: src/parquet/util/rle-test.cc [hunk 1]
```diff
@@ -218,13 +218,13 @@ bool CheckRoundTrip(const vector<int>& values, int bit_width) {
       return false;
     }
   }
-  int encoded_len = encoder.Flush();
+  encoder.Flush();
```
Comment (asandryh) [inline src/parquet/util/rle-test.cc]: Add a check here? `EXPECT_EQ(..., encoder.Flush())`

Comment (wesm) [inline src/parquet/util/rle-test.cc]: The encoded length is data-dependent, but I set it to initialize the decoder with the encoded length

**Comment by wesm** [comment]
green build https://travis-ci.org/wesm/parquet-cpp/builds/112921791

**Comment by julienledem** [comment]
+1

**Pull request closed without merging**